WordPress/src/main/java/org/wordpress/android/ui/qrcodeauth/QRCodeAuthUiStateMapper.kt

```diff
@@ -0,0 +1,73 @@
+package org.wordpress.android.ui.qrcodeauth
+
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.DonePrimaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.DoneSecondaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.AuthenticatingPrimaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.AuthenticatingSecondaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.ErrorPrimaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.ErrorSecondaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.ValidatedPrimaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Action.ValidatedSecondaryAction
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Content
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Error
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Loading
+import org.wordpress.android.ui.qrcodeauth.QRCodeAuthUiState.Scanning
+import javax.inject.Inject
+
+class QRCodeAuthUiStateMapper @Inject constructor() {
+    fun mapLoading() = Loading
```

Comment:
💡 Suggestion: @zwarm Thanks for adding this Mapper class and moving the logic here. I like this implementation. One more thing I would like to improve here is to change the function's name to `mapToLoading` instead of `mapLoading`. The same goes for all the functions. `mapLoading` sounds like we are mapping Loading State. WDYT 🤔

Reply:
@AjeshRPai  - love this suggestion - incorporating it now.